pkg/server/query/entryHandlers.go

```diff
@@ -234,7 +244,7 @@ func pipeSearchWebsocketHandler(myid uint64) func(ctx *fasthttp.RequestCtx) {
 					return
 				}
 			}()
-			pipesearch.ProcessPipeSearchWebsocket(conn, myid, ctx)
+			pipesearch.ProcessPipeSearchWebsocket(conn, orgId, ctx)
```

Comment:
I think now `myid` is never used in the function, so you can also remove it from the definition on line 217